src/charm.py

```diff
@@ -90,9 +95,15 @@ def _configure(self, event: EventBase) -> None:
             self.unit.status = WaitingStatus("Waiting for Multus to be ready")
             event.defer()
             return
+        if not self._relation_created(N2_RELATION_NAME):
```

Comment:
The check that will result in a blocked status should go before the ones that result in a waiting status

Reply:
Done.